ui/src/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { render, screen } from '@testing-library/react';
+import * as Components from '@components';
+import * as OutlineIcons from '@heroicons/react/outline';
+import '@testing-library/jest-dom';
+
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const button = (
```

Comment:
Please move any mocks, variables, helpers etc.. which are part of a specific test suite inside it's own test suite (inside of the "describe" method). Because you may have multiple test suites in a single file, testing different states/props of a component.